tests/src/python/test_qgslayoutexporter.py

```diff
@@ -787,8 +788,16 @@ def testExportToPdfGeoreference(self):
         self.assertAlmostEqual(geoTransform[5], -13.183886222186642, 4)
 
         # check that the metadata has _not_ been added to the exported PDF
+        # starting with Qt 6.9 QPdfWriter generates a pdf with an empty
+        # author by default
         metadata = d.GetMetadata()
-        self.assertNotIn("AUTHOR", metadata)
+        if int(QT_VERSION_STR.split(".")[0]) > 6 or (
```

Comment:
maybe simpler is to do something like ``self.assertEqual(metadata.get("AUTHOR", ""), "")`` which should work on all versions

Reply:
Indeed, this is much simpler. Thanks.